modules/cnf-configuring-workload-hints.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: CONCEPT
+[id="cnf-configuring-workload-hints_{context}"]
+= Configuring workload hints
+
+Workload hints simplify the fine-tuning of performance to industry sector settings. Instead of a “one size fits all” approach, workload hints can cater to usage patterns such as placing priority on:
+
+* Low latency
+* Real-time capability
+* Efficient use of power
+* High throughput
+
+In an ideal world, all of those would be prioritized: in real life, some come at the expense of others. The Node Tuning Operator is now aware of the workload class and better able to meet the demands of a workload class. The cluster admin can now specify into which use case a workload falls. The Node Tuning Operator uses hints fine tune the performance settings.
+
+The environment in which an application is operating influences the choice of hints. In settings such as a core network, only minimal default power tuning is supported using CPU partitioning. In  distributed or Edge settings, latency is a higher priority, but some measures are still taken to optimize power consumption. For far edge settings, at a plane where where software-based radio is transmitted, ultra-low latency is the ultimate priority, and is achieved at the expense of power management.
```

Comment:
> only minimal default power tuning is supported using CPU partitioning
@MarSik  can we state that this is the most common case ?

Reply:
The paragraph needs rewrite, it is jumping from one undefined term to another, mixing concepts etc..

I propose something like this (I edited out all the Telco specific terms that were not defined anywhere):

```suggestion
The environment in which an application is operating influences its behavior. For a typical datacenter with no strict latency requirements, only minimal default tuning is needed that enables CPU partitioning for the few and far between high performance workload pods.
Then there are data centers and workloads where latency is a higher priority, but some measures are still taken to optimize power consumption.
The most complicated cases are clusters close to latency sensitive pieces of equipment like manufacturing machinery, software defined radios etc. This last class of deployment is also known as Far edge. For far edge deployments, ultra-low latency is the ultimate priority, and is achieved at the expense of power management.
```

Reply:
Thanks for the re-write of my first draft. I made some changes as required by our Style Guide.